config/initializers/inflections.rb

```diff
@@ -30,4 +30,5 @@
   inflect.acronym("TSV")
   inflect.irregular("bonus", "bonuses")
   inflect.irregular("info", "info")
+  inflect.irregular("account", "account")
```

Comment:
I presume this is to avoid having to change a thousand account controller references to "accounts"... which sounds wrong anyway, admittedly?  There's just something fishy about summarily deciding that the plural of "account" is no longer "accounts"!!  If we ever introduced the concept have a single user being linked with multiple accounts this could get _really_ confusing! :)

Reply:
Good point. Lemme research what else i could do to keep the controller name singular, that isn't too verbose.

Reply:
Inflection removed, turns out it only mattered in routes. 

The workaround is
```ruby
resource :account, controller: "account" do
  stuff
end
```

Reply:
Ah!  That _is_ much better.  Hope that feels worthwhile to you, too.